mmv1/third_party/terraform/services/compute/resource_compute_instance_group_manager.go.erb

```diff
@@ -216,6 +216,48 @@ func ResourceComputeInstanceGroupManager() *schema.Resource {
 				},
 			},
 
+      <% unless version == "ga" -%>
```

Comment:
here it still looks not aligned with the other parts of this file. it needs a bunch of spaces/tabs in all these lines you changed with the new fields